enhancements/console/custom-logos.md

```diff
@@ -0,0 +1,147 @@
+
+To address this, we need to allow users to add custom logos for both the masthead and favicon that are compatible with light and dark themes. This ensures that the logos are always visible and aesthetically pleasing, regardless of the theme mode.
+
+The custom logos feature will enable users to specify different logos for the masthead and favicon based on the theme mode. This will involve exposing a new API endpoint to support custom logos for both light and dark themes.
+
+## Motivation
+
+The OpenShift Container Platform (OCP) web console is currently being upgraded to PatternFly 6. In PatternFly 6, the masthead color changes based on the mode (light or dark). Consequently, a single custom logo may not be suitable for both theme modes. This is evident with the existing OKD and OpenShift logos, which assume a dark masthead background and include white text.
+
+### User Stories
+
+* As an OpenShift web console user, I want to be able to add custom logos for light and dark theme modes in the masthead and favicon.
+
+### Goals
+
+This feature should allow users to add custom logos for the masthead and favicon that are compatible with both light and dark theme modes in the OpenShift web console.
+
+### Non-Goals
+
+
+## Proposal
+
+### Workflow Description
+
+├── spec
+│   ├── customization
+│       ├── customLogos
+│           ├── type
+│           ├── themes
+│               ├── type
+│               ├── file
+│                   ├── key
+│                   ├── name
+└── ...
+
+### Risks and Mitigations
+
+1. Users could set both the `CustomLogoFile` and `CustomLogos` APIs. The `CustomLogos` API configuration will take precedence over the old `CustomLogoFile` field.
+
+2. If only the dark or light theme is specified, the default OCP logo will be used for the unspecified theme.
+
+3. Handling different custom logos for light and dark themes may increase the complexity of the backend and UI logic. Tests are added to validate the new  logic and a fallback is implemented to default to the OCP logo if any theme is missing.
+
+4. Users might experience confusion with the introduction of new logo configuration options, especially if they are familiar with the old method, which may soon be deprecated. Provide a comprehensive documentation and tools that guide users through the transition. Include clear instructions about the changes and their benefits.
+
+
+### Drawbacks
+
+N/A
+
+## API Design Details
+
+The configuration for custom logos will include support for both masthead and favicon types, with separate files for light and dark themes:
+
+```yaml
+customLogos:
+  - type: Masthead
+    themes:
+      - type: Light
+        file:
+          key: logo-light.svg
+          name: masthead-logo-light
+      - type: Dark
+        file:
+          key: logo-dark.svg
+          name: masthead-logo-dark
+  - type: Favicon
+    themes:
+      - type: Light
+        file:
+          key: favicon-light.ico
```

Comment:
lets use .png in the example since we are recommending it in the API change.
```suggestion
          key: favicon-light.png
```

also below